### PR TITLE
[DOCS] Rules page UI updates

### LINKED ISSUE #1382: [DOCS] Rules page UI updates
## Description 

The UI for the Rules page has changed in 8.0 to be more compact. This affects both the **Rules** tab and the **Rule Monitoring** tab. Also, the **Rule Monitoring** tab has new functionality to allow more interaction with rules.

## Rule tab

For details, see this issue (possibly others):

- https://github.com/elastic/kibana/issues/117460

### Sample screenshots - Rule tab

OLD rules table (7.17):
image

NEW rules table (8.0):
image

## Rule Monitoring tab

For details, see:

- https://github.com/elastic/kibana/pull/119644 — This PR addresses these issues:
   - https://github.com/elastic/security-team/issues/2073
   - https://github.com/elastic/security-team/issues/2074

### Sample screenshots - Rule Monitoring tab

OLD rule monitoring table (7.17):
image

NEW rule monitoring table (8.0):
image

## Notes

- As always, look for ways to reduce future maintenance of any images/animations involving the Rules table.

### PR DESCRIPTION
**Pull request by joepeeples**
* Resolves #1382, regarding UI changes to both the **Rules** and **Rule Execution** tabs on the Rules page.

Previews:
* [Elastic Security UI | Rules page](https://security-docs_1450.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html#rules-page)
* [Monitor and troubleshoot rule executions](https://security-docs_1450.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-monitor.html)

### CODE REVIEW

**Review by nastasha-solomon — APPROVED** [review]

File: docs/detections/rules-ui-monitor.asciidoc [hunk 1]
```diff
@@ -9,8 +9,7 @@ times, select the *Rule Monitoring* tab on the *Rules* page (*Detect* ->
 [role="screenshot"]
 image::images/monitor-table.png[]
 
-For detailed information on a rule, its generated alerts, and its errors, click on
-a rule name in the *All rules* table.
+For detailed information on a rule, its generated alerts, and its errors, click on a rule name in the *All rules* table. You can also perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
```
Comment (benironside) [inline docs/detections/rules-ui-monitor.asciidoc:12]:
```suggestion
For detailed information on a rule, alerts it generated, and associated errors, click on its name in the *All rules* table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
```
Nits. Mostly I just thought that "its generated alerts" was an awkward turn of phrase.

**Review by benironside — APPROVED** [review]

File: docs/detections/rules-ui-monitor.asciidoc (hunk 1 above)
Comment (joepeeples) [reply docs/detections/rules-ui-monitor.asciidoc:12]: Agree, with a couple slight amendments:
```suggestion
For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
```

**Review by jmikell821 — APPROVED** [review]
Agree w/ yours and Ben's suggestions for that one comment. LGTM otherwise.

File: docs/detections/rules-ui-monitor.asciidoc [hunk 2]
```diff
@@ -9,8 +9,7 @@ times, select the *Rule Monitoring* tab on the *Rules* page (*Detect* ->
 [role="screenshot"]
 image::images/monitor-table.png[]
 
-For detailed information on a rule, its generated alerts, and its errors, click on
-a rule name in the *All rules* table.
+For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
```
Comment (vitaliidm) [inline docs/detections/rules-ui-monitor.asciidoc:12]: To me, it looks like, `This allows also..` in 
> For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. This also allows you to perform the same actions...

refers more to what is possible after clicking on rule's name rather that what is possible to do on monitoring table.
Maybe, worth to highlight actions are possible on monitoring table?

Comment (joepeeples) [reply docs/detections/rules-ui-monitor.asciidoc:12]: @vitaliidm I agree. We could reorder and rephrase slightly to lead with those actions:
```suggestion
You can perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules. For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. 
```

**Review by jethr0null — COMMENTED** [review]
LGTM!

File: docs/detections/rules-ui-monitor.asciidoc (hunk 2 above)
Comment (vitaliidm) [reply docs/detections/rules-ui-monitor.asciidoc:12]: Thanks, @joepeeples. Looks good

**Review by vitaliidm — APPROVED** [review]
LGTM

**Pull request merged**